### PR TITLE
docs: fix SS06, PR08, RT02, RT03, SA04 in pandas.DatetimeIndex.indexe…

### LINKED ISSUE #8: Fix SS06 errors in pandas.DatetimeIndex.indexer_at_time

### PR DESCRIPTION
**Pull request by rahmadiantio**
…r_at_time

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

 Fix : https://github.com/pandanistas/pandanistas_sprint_ui2020/issues/8

### CODE REVIEW

File: pandas/core/indexes/datetimes.py [hunk 1]
```diff
@@ -702,23 +702,27 @@ def inferred_type(self) -> str:
...
+            A datetime.time or string in appropriate format ("%H:%M", "%H%M",
             "%I:%M%p", "%I%M%p", "%H:%M:%S", "%H%M%S", "%I:%M:%S%p",
             "%I%M%S%p").
+        asof : bool
 
         Returns
         -------
-        values_at_time : array of integers
+        array of integers
+            An array is a container object that holds a fixed number of
```
Comment (Iqrar99) [inline pandas/core/indexes/datetimes.py]: This line has a trailing space. Should be removed.
```suggestion
            An array is a container object that holds a fixed number of
```

File: pandas/core/indexes/datetimes.py [hunk 2]
```diff
@@ -702,23 +702,27 @@ def inferred_type(self) -> str:
...
         -------
-        values_at_time : array of integers
+        array of integers
+            An array is a container object that holds a fixed number of 
+            values of a single type.
 
         See Also
         --------
-        indexer_between_time, DataFrame.at_time
+        indexer_between_time : Return index locations of values between
```
Comment (Iqrar99) [inline pandas/core/indexes/datetimes.py]: Also this line.
```suggestion
        indexer_between_time : Return index locations of values between
```

**Review by Iqrar99 — CHANGES_REQUESTED** [review]
Some trailing spaces should be removed.

File: pandas/core/indexes/datetimes.py [hunk 3]
```diff
@@ -702,23 +702,27 @@ def inferred_type(self) -> str:
...
-            datetime.time or string in appropriate format ("%H:%M", "%H%M",
+            A datetime.time or string in appropriate format ("%H:%M", "%H%M",
             "%I:%M%p", "%I%M%p", "%H:%M:%S", "%H%M%S", "%I:%M:%S%p",
             "%I%M%S%p").
+        asof : bool
 
         Returns
         -------
-        values_at_time : array of integers
+        array of integers
```
Comment (datapythonista) [inline pandas/core/indexes/datetimes.py]: Isn't this returning an Index?

File: pandas/core/indexes/datetimes.py [hunk 4]
```diff
@@ -702,23 +702,27 @@ def inferred_type(self) -> str:
...
+        Return the index location at certain times of the day (eg 9:30 am).
 
         Parameters
         ----------
         time : datetime.time or str
-            datetime.time or string in appropriate format ("%H:%M", "%H%M",
+            A datetime.time or string in appropriate format ("%H:%M", "%H%M",
             "%I:%M%p", "%I%M%p", "%H:%M:%S", "%H%M%S", "%I:%M:%S%p",
             "%I%M%S%p").
+        asof : bool
```
Comment (datapythonista) [inline pandas/core/indexes/datetimes.py]: Do you mind adding a description here?

**Review by datapythonista — COMMENTED** [review]
Thanks for the contribution @rahmadiantio, do you have time to address the comments?

**Comment by datapythonista** [comment]
Closing as stale. @rahmadiantio let us know if you finally want to address the review comments.

**Pull request closed without merging**